components/footer/footer.scss

```diff
@@ -4,3 +4,18 @@ footer {
   bottom: 0;
   left: 0;
 }
+
+.site-footer {
+  background-color: $black;
+
+  .link-list {
+    @media (min-width: $bp-md) {
+      column-count: 2;
+      break-inside: avoid-column;
+    }
+  }
+
+  hr {
+    border-color: #ccc;
```

Comment:
Please only use color variables for any colours (in this case`$gray-20`). I'm gonna add stylelint & prettier to this repo soon so things are more aligned with the rules we set for Foundation site.